src/settingsmanager.cpp

```diff
@@ -9,6 +9,11 @@
 
 QString getDataDirectory()
 {
+    auto currentDataDir = QString::fromStdString(kiwix::removeLastPathElement(kiwix::getExecutablePath()));
+    auto portableFile = QFileInfo(currentDataDir, ".portable");
+    if (portableFile.exists())
+        return currentDataDir + "/data";
```

Comment:
Do we (intend to) provide a portable package for Windows? If yes, then the directory separator should depend on the the platform.